static/erdblick/view.js

```diff
@@ -195,17 +197,20 @@ export class ErdblickView
         // Grow the viewport rectangle by 25%
         let expandLon = sizeLon * 0.25;
         let expandLat = sizeLat * 0.25;
-        if (west > east) {
-            sizeLon += 360.;
-        }
-
-        let viewport = new MapViewerViewport(south, west, sizeLon, sizeLat, centerLon, centerLat, this.viewer.camera.heading);
-        this.model.setViewport(viewport);
+        this.model.setViewport({
+            south: south - expandLat,
+            west: west - expandLat,
```

Comment:
```suggestion
            west: west - expandLon,
```
And do we not need to worry about the value wrapping around somehow?

Reply:
Nope, the AABB class takes care of that.